sdk/python/foundation-models/system/inference/fill-mask/fill-mask-batch-endpoint.ipynb

```diff
@@ -0,0 +1,483 @@
+    "* Connect to AzureML Workspace. Learn more at [set up SDK authentication](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk). Replace  `<WORKSPACE_NAME>`, `<RESOURCE_GROUP>` and `<SUBSCRIPTION_ID>` below.\n",
+    "* Connect to `azureml` system registry.\n",
+    "* Create or update compute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import packages used by the following code snippets\n",
+    "import csv\n",
+    "import json\n",
+    "import os\n",
+    "import random\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import pandas as pd\n",
+    "import urllib.request\n",
+    "\n",
+    "from azure.ai.ml import Input, MLClient\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.ai.ml.entities import (\n",
+    "    AmlCompute,\n",
+    "    BatchDeployment,\n",
+    "    BatchEndpoint,\n",
+    "    BatchRetrySettings,\n",
+    "    Model,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subscription_id = \"<SUBSCRIPTION_ID>\"\n",
+    "resource_group_name = \"<RESOURCE_GROUP>\"\n",
+    "workspace_name = \"<WORKSPACE_NAME>\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Connect to workspace and registry using ML clients."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "workspace_ml_client = MLClient(\n",
+    "        credential,\n",
+    "        subscription_id=subscription_id,\n",
+    "        resource_group_name=resource_group_name,\n",
+    "        workspace_name=workspace_name\n",
+    ")\n",
+    "# the models, fine tuning pipelines and environments are available in the AzureML system registry, \"azureml\"\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a compute cluster.\n",
+    "Use the model card from the AzureML system registry to check the minimum required inferencing SKU, referenced as `size` below. If you already have a sufficient compute cluster, you can simply define the name in `compute_name` in the following code block. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compute_name = \"cpu-cluster\""
+   ]
+  },
+  {
```

Comment:
How about adding a cell here that says something like this:

Otherwise, create a new compute cluster. Set the 'size' to satisfy the minimum required inferencing SKU.